pkg/lang/frontend/starlark/v0/config/config.go

```diff
@@ -95,18 +95,19 @@ func ruleFuncJupyter(thread *starlark.Thread, _ *starlark.Builtin,
 func ruleFuncPyPIIndex(thread *starlark.Thread, _ *starlark.Builtin,
 	args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
 	var url, extraURL starlark.String
+	var trust starlark.Bool
 
 	if err := starlark.UnpackArgs(rulePyPIIndex, args, kwargs,
-		"url?", &url, "extra_url?", &extraURL); err != nil {
+		"url?", &url, "extra_url?", &extraURL, "trust", &trust); err != nil {
```

Comment:
This should be optional.
```suggestion
		"url?", &url, "extra_url?", &extraURL, "trust?", &trust); err != nil {
```

Reply:
done